torch/onnx/_internal/exporter.py

```diff
@@ -1190,16 +1190,21 @@ def __init__(
             self._assert_fake_tensor_mode()
 
     def export(self) -> ONNXProgram:
+        from torch.export._trace import DEFAULT_EXPORT_DYNAMO_CONFIG
+
         # TODO: Defer `import onnxscript` out of `import torch` path
         # https://github.com/pytorch/pytorch/issues/103764
         from torch.onnx._internal.fx import decomposition_skip
 
         with self.options.diagnostic_context, decomposition_skip.enable_decomposition_skips(
             self.options
         ):
-            graph_module = self.options.fx_tracer.generate_fx(
-                self.options, self.model, self.model_args, self.model_kwargs
-            )
+            with torch._dynamo.config.patch(
+                dataclasses.asdict(DEFAULT_EXPORT_DYNAMO_CONFIG)
```

Comment:
This might introduce more things than we want. 

It is ok for now. Right now it only patches 'print' support, and `allow_rnn=True`.